docs/articles/case-study-gdpr-it.md

```diff
@@ -0,0 +1,94 @@
+---
+title: 'Case study: garantire il diritto all''oblio'
+created: '2020-11-13T16:43:39.086Z'
+modified: '2020-11-14T09:47:45.393Z'
+---
+
+# Case study: garantire il diritto all'oblio
+
+Quando progettiamo una nuova funzionalità per l'app IO, la più grande sfida che dobbiamo affrontare è quella di saper coniugare un'esperienza utente morbida con elevati requisiti di sicurezza, un'implementazione lineare con la complessità del contesto giuridico in cui IO si colloca rispetto al Cittadino. Questo richiede forte collaborazione tra sviluppatori, service designer, legali, esperti di sicurezza e di operations.
+
+Questo processo interdisciplinare è stato messo particolarmente alla prova per recepire le norme del GDPR e integrarne i principi all'interno dell'app IO. In particolare, ai Cittadini deve essere garantito il _diritto all'oblio_, cioè deve essere rispettata la loro eventuale decisione di rimuovere i propri dati personali che vengono conservati dall'app IO.
+
+In questo articolo vogliamo condividere le soluzioni tecniche messe in campo per garantire il diritto all'oblio del Cittadino. In particolare vorremmo condividere l'utilizzo di Azure Durable Functions, e di come abbiamo tratto veramente grande beneficio da questo strumento.
+
+
+## Specifiche
+L'implementazione che andremo a presentare è stata pensata per soddisfare i seguenti requisiti:
+* Come Cittadino voglio poter richiedere la cancellazione dei miei dati.
+* Come Cittadino voglio poter ritirare la precedente richiesta entro 7gg.
+* Come DPO voglio che sia tenuta traccia di tutte le richieste del Cittadino e dello stato di lavorazione delle stesse.
+* Come Cittadino voglio essere notificato dell'avvenuta cancellazione
+* Come DPO voglio mantenere i dati del Cittadino per gestire eventuali contestazioni o contenziosi
+
+
+
+## Implementazione
+### Task
```

Comment:
Possiamo linkare le stories su pivotal?

Reply:
questi task sono una rielaborazione a posteriori, non abbiamo realmente lavorato in maniera così ben definita. Però sicuramente possiamo linkare la epic che raccoglie tutti i task